### PR TITLE
Weel name fix parse header

### PR DESCRIPTION
**Pull request by Czaki**
Fixing wheel name with parsing header from python code (based on pypa/wheel#314) and calculate proper platform tag.

 - [x] parsing headers to extract minimal version
 - [x] test of added functions 
 - [ ] new test for `delocate_wheel`

### CODE REVIEW

File: delocate/delocating.py [hunk 1]
```diff
@@ -401,13 +411,99 @@ def delocate_wheel(in_wheel,
                 set_install_id(copied_path, install_id_root + lib_base)
                 validate_signature(copied_path)
             _merge_lib_dict(all_copied, copied_libs)
+        if check_wheel_name or fix_wheel_name:
+            wheel_name = os.path.basename(in_wheel)
+            new_name = update_wheel_name(wheel_name, wheel_dir)
+            if check_wheel_name and new_name != wheel_name:
+                raise DelocationError("Wheel name do not satisfy minimal package requirements")
```
Comment (radarhere) [inline delocate/delocating.py:418]:
```suggestion
                raise DelocationError("Wheel name does not satisfy minimal package requirements")
```

File: delocate/parse_macho_header.py [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+                version, arch =  read_mach_header(lib_file, 0)
+                return {arch: version}
+            except ValueError:
+                """when some error during read library files"""
+                return {}
+
+
+def read_mach_header(lib_file, seek=None):
+    """
+    This funcition parse mach-O header and extract
```
Comment (radarhere) [inline delocate/parse_macho_header.py:294]:
```suggestion
    This function parses mach-O header and extracts
```

**Comment by matthew-brett** [comment]
Hi @Czaki  - you're welcome to log into my machine at home if you want.  Can you send your ssh public key to me via email? - matthew.brett at gmail

**Comment by Czaki** [comment]
I have sent mail.

**Comment by HexDecimal** [comment]
Resolved by #198

**Pull request closed without merging**